docs/capabilities.md

```diff
@@ -2,16 +2,54 @@
 
 ## Capabilities
 
+Before we can send any command to the browser we drive we need to create a [WebDriver](https://www.w3.org/TR/webdriver) session.
+This should be always the first step of interaction using the protocol.
+To create a new session we need to provide some requirements like `browserName`, `browserVersion`, `platformName`.
+These requirements are called [`Capabilities`](https://developer.mozilla.org/en-US/docs/Web/WebDriver/Capabilities).
+This is espetially important when we interact not directly with a single browser but with a grid of browsers.
+Using `alwaysMatch` property we can provide some strict requirements.
+If the WebDriver sever can`t provide the required features the session creation will fail.
+Using `firstMatch` property we can provide a list of requirements.
+During session creation the server will pick the first capability it's able to match.
+`firstMatch` can of course be combined with `alwaysMatch` to narrow down the selection.
+
 **PROPERTIES**
 
 - `alwaysMatch?`: [BrowserCapability](#browsercapability)
 - `firstMatch?`: [BrowserCapability](#browsercapability)[]
 
+**EXAMPLES**
+
+```typescript
+const capabilities = {
+  alwaysMatch: {
+    browserName: 'firefox',
+    browserVersion: '60'
+  }
+}
+```
+
+```typescript
+const capabilities = {
+  firstMatch: [
+    { browserName: 'chrome' },
+    { browserName: 'firefox' }
+  ]
+}
+```
+
+**SEE ALSO**
+
+- [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/WebDriver/Capabilities)
+
 ## BrowserCapability
 
 **PROPERTIES**
 
-- `browserName`: string
+- `browserName?`: string
+- `browserVersion?`: string
```

Comment:
Will these props be retained when we generate the docs again? Maybe when we add more apis are something.